core/src/main/scala/com/dimafeng/testcontainers/DockerComposeContainer.scala

```diff
@@ -162,6 +162,7 @@ class DockerComposeContainer(composeFiles: ComposeFile,
 
     container.withPull(pull)
     container.withLocalCompose(localCompose)
+    container.withOptions("--compatibility")
```

Comment:
I'm wondering if there are any downsides to have this enabled by default -- will it work in all cases? Would it make sense to make it a constructor parameter or allow to pass "options"

Reply:
So, as far as I read, we ensure v1 compatibility (nothing will break there) but not completely v2 (issue with underscore and dash) https://docs.docker.com/compose/cli-command-compatibility/#flags-that-will-not-be-implemented

So I believe this change is safe.